lib/middleware/parameter.js

```diff
@@ -232,8 +232,8 @@ module.exports = async (ctx, next) => {
             // opencc
             if (ctx.query.opencc) {
                 ctx.state.data.item.forEach((item) => {
-                    item.title = simplecc(item.title, ctx.query.opencc);
-                    item.description = simplecc(item.description, ctx.query.opencc);
+                    item.title = simplecc(item.title || '', ctx.query.opencc);
+                    item.description = simplecc(item.description || item.title || '', ctx.query.opencc);
```

Comment:
别置空，直接抛可读性的错，置空会推送错误 item

Reply:
Updated with
```javascript
item.title = simplecc(item.title ?? item.link, ctx.query.opencc);
item.description = simplecc(item.description ?? item.title ?? item.link, ctx.query.opencc);
```
because in some routes, e.g. `/coolapk/huati/%E5%A5%BD%E7%94%B5%E5%BD%B1?opencc=s2t` we only have item.link, both item.title and item.description are undefined.
Throwing an exception due to error in only one feed item doesn't seems a robust solution.

<details>

<summary>original feed screenshot</summary>

![1](https://user-images.githubusercontent.com/11386903/132730746-e8fb05d1-110c-411c-89ac-6bc176fda899.PNG)

</details>